python/az/aro/azext_aro/aaz/__init__.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
This looks like an inadvertent application of the RP's license to Azure CLI code.

If I'm understanding this correctly, this PR should not be changing any Python code, so this entire folder can be backed out.

Reply:
reverted these changes